segunda_edicion/proxectos/pedro-luis/env_var.py

```diff
@@ -0,0 +1,5 @@
+url = 'https://api.tago.io/data'
+headers = {'Device-Token': '<yourToken>',
+           'Content-Type': 'application/json'}
+SSID = "A Industriosa"
+PASS = "alfageme"
```

Comment:
Quizáis deixar aquí unas variables de exemplo, tipo `<wifi_ssid>` e `<contrasinal_wifi>`. Sei que non hai nada relevante na wifi pero bueno nunca está de máis 😄